WordPress/Classes/Extensions/Media+Blog.swift

```diff
@@ -0,0 +1,38 @@
+import Foundation
+
+extension Media {
+
+    /// Inserts and returns a new managed Media object, in the context.
+    ///
+    class func inserted(into context: NSManagedObjectContext) -> Media {
+        let media = NSEntityDescription.insertNewObject(forEntityName: "Media", into: context) as! Media
+        media.creationDate = Date()
+        media.mediaID = 0
+        return media
+    }
+
+    /// Inserts and returns a new managed Media object, with a blog.
+    ///
+    class func insertedWith(blog: Blog) -> Media {
```

Comment:
Since the `managedObjectContext` would only be `nil` if the blog object was deleted, according to the docs, I think force unwrapping and incurring a crash would be ideal for debugging? Since we'd be in a pretty bad state if we're trying to insert media with a deleted blog object.

Or we could go ahead and add the extra overhead via `throws` to be cautious from the ground up. Thoughts?